.github/workflows/build_and_test.yml

```diff
@@ -5406,6 +5329,12 @@ jobs:
     steps:
     - name: install xerces
       run: sudo apt-get -y install libxerces-c-dev
+    - name: update apt
+      run: sudo apt-get update
```

Comment:
Move this up ahead of installing xerces-c.
Consider combining the install steps to potentially save time.